README.md

```diff
@@ -50,4 +50,39 @@ To manually read data from a topic, run this command:
 
 To exit, press `Ctrl+C`
 
-More info [here](https://docs.confluent.io/platform/current/app-development/kafkacat-usage.html#consumer-mode)
\ No newline at end of file
+More info [here](https://docs.confluent.io/platform/current/app-development/kafkacat-usage.html#consumer-mode)
+
+## Connection problem in AKHQ
+Depending on you docker version, you may be experiencing some connectivity issue in AKHQ to Kafka.
+Please verify first that both Zookeeper and Kafka containers are up. If it is so, you need to do an update to the docker-compose.yaml file. 
+
+You must update the file from : 
+```
+  akhq:
+    environment:
+      AKHQ_CONFIGURATION: |
+        akhq:
+          connections:
+            docker-kafka-server:
+              properties:
+                bootstrap.servers: "cloudmc-monetization-local-kafka-1:9092"
+```
+to :
+```
+  akhq:
+    environment:
+      AKHQ_CONFIGURATION: |
+        akhq:
+          connections:
+            docker-kafka-server:
+              properties:
+                bootstrap.servers: "cloudmc-monetization-local_kafka_1:9092"
+```
+
+Then run again the docker-compose command.
+```
+docker-compose up -d
+```
+
+
+It seems the docker container name generation changed in the most recent version of docker. They use now hyphens instead of underscores.
```

Comment:
```suggestion
It seems the docker container name generation changed in the most recent version of docker. They now use hyphens instead of underscores.
```